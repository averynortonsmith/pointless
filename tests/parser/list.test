
x = []
y = [1]
z = [1, 2, 3]
a = [1, 2, 3,]