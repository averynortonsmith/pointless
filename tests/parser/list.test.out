(Program 
 null
 []
 (Def 
  (Name "x")
  (List []))
 (Def 
  (Name "y")
  (List (Number 1)))
 (Def 
  (Name "z")
  (List 
   (Number 1)
   (Number 2)
   (Number 3)))
 (Def 
  (Name "a")
  (List 
   (Number 1)
   (Number 2)
   (Number 3))))
